manifests/package.pp

```diff
@@ -45,14 +45,20 @@
       $repo_require = undef
 
       $pkg_version = inline_template("<%= scope.lookupvar('sensu::version').sub(/(.*)\\-/, '\\1.') %>")
-      $pkg_title = 'Sensu'
-      $pkg_name = 'Sensu'
-      $pkg_source = "C:\\Windows\\Temp\\sensu-${sensu::version}.msi"
-      $pkg_require = "Remote_file[${pkg_source}]"
+      # Download the latest published version
+      $pkg_url_version = $::sensu::version ? {
+        installed => 'latest',
+        default   => $pkg_version,
+      }
```

Comment:
@agoddard said he would create it manually and talk to the dev team to get it part of the release process